server/meshmodel/argo-cd/relationships/hierarchical_inventory.json

```diff
@@ -0,0 +1,45 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Hierarchical",
+  "metadata": {
+    "description": "A parent-child relationship implies the requirement of the parent component before the child component can be created"
```

Comment:
@innocentrda please take a quick moment to punctuate the first sentence and to include a second sentence using the pod and namespace example that has been suggested. Look to other relationships for text to copy/paste. There might be one of these already written.

Reply:
We're missing a relationship-specific description. Will you re-add this so that users of this relationship have a description specific to AppSet and App?

Reply:
@leecalcote  The description is updated accordingly. Is it much better?